graphql-async/test/jbuild

```diff
@@ -0,0 +1,10 @@
+(jbuild_version 1)
+
+(executables
+ ((libraries (graphql_async alcotest async))
```

Comment:
I think it's fine to include #48 in this PR (2a6988ec83a8085079a90e6139b356e919d6f271), otherwise Travis will just fail on that.